src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -196,6 +196,10 @@
 
   static final String UNDEFINED_LABEL = "undefined label \"%s\"";
 
+  static final String EXPOSE_DEPRECATED_WARNING =
```

Comment:
I've tried both in the `@fileoverview` block and at a function. Neither worked. I got the same behavior with the "nonStandardJsDocs" group.

Test case:
```JavaScript
/**
 * @fileoverview
 * @suppress {nonStandardJsDocs}
 */
function f1() {
  /**
   * @foo
   * @expose
   */
  window.x = true;
}
```

Both work when using the `--jscomp_off` flag.